FinalProject/client/app/src/main/java/com/runbuddy/LoginActivity.java

```diff
@@ -0,0 +1,135 @@
+package com.runbuddy;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.JsonObjectRequest;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.firebase.iid.FirebaseInstanceId;
+import com.google.firebase.iid.InstanceIdResult;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class LoginActivity extends AppCompatActivity {
+
+    String token = "";
+    public static TextInputEditText userName;
+    TextInputEditText password;
+    Button next;
+    Button cancel;
+    private RequestQueue _queue;
+    private static final String ACTIVITY_TAG = "login";
+    private static final String SERVER_ADDRESS = "http://10.0.0.25:8080/";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+
+        userName = (TextInputEditText)findViewById(R.id.userNameEditText);
+        password = (TextInputEditText)findViewById(R.id.repasswordEditText);
+        next = (Button)findViewById(R.id.nextButton);
+        next.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                LogUserIn(userName.getText().toString(), password.getText().toString());
+            }
+        });
+
+        cancel = (Button)findViewById(R.id.cancelButton);
+        cancel.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent cancelIntent = new Intent(LoginActivity.this, MainActivity.class);
+                startActivity(cancelIntent);
+            }
+        });
+    }
+
+    /**********************************************************************
+        This method checks the users login credentials and logs him
+        in if he is in the system
+    ***********************************************************************/
+    private void LogUserIn(String email, String password){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.